public/index.html

```diff
@@ -1,30 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="utf-8" />
-        <link rel="icon" href="%PUBLIC_URL%/logoActive.svg" />
-        <!-- should probably be replaced -->
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <meta name="theme-color" content="#000000" />
-        <meta
-            name="description"
-            content="Progressive web app for finding the perfect movie to watch with a friend."
-        />
-        <meta name="author" content="" />
-        <meta property="og:type" content="web app" />
-        <meta property="og:image" content="%PUBLIC_URL%/logoActive.svg" />
-        <!-- should probably be replaced -->
-        <meta property="og:image:type" content="image/svg" />
-        <meta property="og:image:width" content="825" />
-        <meta property="og:image:height" content="825" />
-        <link rel="apple-touch-icon" href="%PUBLIC_URL%/logoActive.svg" />
-        <!-- should probably be replaced -->
-        <!--
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" href="%PUBLIC_URL%/Icon.svg" />
```

Comment:
it looks like I changed a ton because my margins are slightly different.  But I just changed the icon back.